### ISSUE TITLE
About the basis of the division of time periods between the two markets (up and down)

### ISSUE THREAD

**Issue by lanqian110** [SteamerLee/RiPO#1]
Thanks for your contribution.
I would like to know what is the basis for the division of time periods between market1 and market2 in the config.py file, is it by index? Because I'm a bit confused about the time period of the training set, validation set and test set...
Looking forward to your valuable reply, thank you very much!

**Comment by SteamerLee** [comment]
Hi, the settings of two datasets are based on including the pandemic period or not. You may refer to the Section 5.1 in the paper for more details. Thank you for your kind attention!
